lib/network/modules/LayoutEngine.js

```diff
@@ -34,7 +34,6 @@ import TimSort from 'timsort';
 import util from 'vis-util';
 import NetworkUtil from '../NetworkUtil';
 import { HorizontalStrategy, VerticalStrategy } from './components/DirectionStrategy.js';
-import { fillLevelsByDirection } from './layout-engine'
```

Comment:
To be honest I would have expected to see any changes to the actual lib here. Why are these changes needed to fix the examples overview?